{"uuid": "3ea9f08a-b2ae-4a3d-8d3e-c2b2edcfc89a", "children": ["fe8ed084-62cd-4635-b82d-5a92ef333390"], "befores": [{"name": "login_pur", "status": "broken", "statusDetails": {"message": "Exception: 元素未找到\n", "trace": "  File \"c:\\programs\\python\\python36\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\programs\\python\\python36\\lib\\site-packages\\_pytest\\fixtures.py\", line 918, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\programs\\python\\python36\\lib\\site-packages\\_pytest\\fixtures.py\", line 778, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"C:\\gitStore\\test_zjc_pytest\\case\\zjc\\conftest.py\", line 6, in login_pur\n    login_page.pur_login(user,psw)\n  File \"C:\\gitStore\\test_zjc_pytest\\pageObject\\zjc\\loginPage.py\", line 30, in pur_login\n    self.u_input(username)\n  File \"C:\\gitStore\\test_zjc_pytest\\pageObject\\zjc\\loginPage.py\", line 20, in u_input\n    self.send_keys(self.selector_user,username)\n  File \"C:\\gitStore\\test_zjc_pytest\\public\\basePage.py\", line 102, in send_keys\n    element=self.find_element(selector)\n  File \"C:\\gitStore\\test_zjc_pytest\\public\\basePage.py\", line 68, in find_element\n    raise Exception('元素未找到')\n"}, "start": 1566982777187, "stop": 1566982789009}], "start": 1566982777187, "stop": 1566982789154}